lib/qmi/codec/network_access.ex

```diff
@@ -40,14 +40,19 @@ defmodule QMI.Codec.NetworkAccess do
   @type network() :: :network_unknown | :network_3gpp2 | :network_3gpp
 
   @type serving_system_indication() :: %{
-          name: :serving_system_indication,
-          service_id: 0x03,
-          indication_id: 0x0024,
-          serving_system_registration_state: serving_system_registration_state(),
-          serving_system_cs_attach_state: attach_state(),
-          serving_system_ps_attach_state: attach_state(),
-          serving_system_selected_network: network(),
-          serving_system_radio_interfaces: [radio_interface()]
+          required(:name) => :serving_system_indication,
+          required(:service_id) => 0x03,
+          required(:indication_id) => 0x0024,
+          required(:serving_system_registration_state) => serving_system_registration_state(),
+          required(:serving_system_cs_attach_state) => attach_state(),
+          required(:serving_system_ps_attach_state) => attach_state(),
+          required(:serving_system_selected_network) => network(),
+          required(:serving_system_radio_interfaces) => [radio_interface()],
+          optional(:cell_id) => integer(),
+          optional(:timezone_offset) => integer(),
```

Comment:
The timezone offset below is multiplied by 15/60, so it's a float. However, I'm wondering if would just be better to keep it an integer and multiply by 15 so that it's in minutes.

Reply:
The reason I when with hours is because most time zones indicated in hours. `UTC-7` for example. But I can which it to minutes if you'd like?

Reply:
I also change my mind on this to be seconds so that it can be a [`Calendar.utc_offset/0`](https://hexdocs.pm/elixir/Calendar.html#t:utc_offset/0).